cves/kernel/CVE-2022-29156.yml

```diff
@@ -360,8 +397,10 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
```

Comment:
I like that you added the last sentence stating that the problem really was super small and just accidentally doing the same thing one too many times.